src/generate-from-template.js

```diff
@@ -32,8 +33,16 @@ const generateFromTemplate = async ({ templateName }) => {
   // eslint-disable-next-line import/no-extraneous-dependencies,global-require,import/no-dynamic-require
   const templatePackage = require(templatePackageName);
 
+  let templateBanner;
+  if (typeof templatePackage.getTemplateBanner === 'function') {
+    const templateBannerResponse = templatePackage.getTemplateBanner(kleur);
```

Comment:
Do we want to consider forcing this function to be async ? That way the banner has the ability to be a bit dynamic if the user wants ( like fetching data to display ).

Reply:
I actually intentionally left it sync, since this is in step 1, which is really just meant to import the template, I didnt want templates to get heavy handed with what they display